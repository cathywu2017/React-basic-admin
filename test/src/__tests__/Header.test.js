import React from 'react';
import {
  cleanup,
  render,
} from '@testing-library/react';
import Header from '../Header';

afterEach(cleanup);

it('renders Header correctly', () => {
  const { container } = render(
    <Header />,
  );

  expect(container).toMatchSnapshot();
});
