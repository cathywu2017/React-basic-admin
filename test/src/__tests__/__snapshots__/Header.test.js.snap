// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header correctly 1`] = `
<div>
  <div
    class="ui inverted menu"
  >
    <a
      class="header item"
    >
      Header
    </a>
  </div>
</div>
`;
